website/screens/components/alert/specs/AlertSpecsPage.tsx

```diff
@@ -0,0 +1,541 @@
+import Image from "next/image";
+import AlertSpecsImage from "./images/alert_specs.png";
+import { DxcLink, DxcList, DxcStack, DxcTable, DxcText } from "@dxc-technology/halstack-react";
+import Code from "../../../common/Code";
+import DocFooter from "../../../common/DocFooter";
+import HeadingLink from "../../../common/HeadingLink";
+import Figure from "../../../common/Figure";
+import AlertAnatomyImage from "./images/alert_anatomy.png";
+import AlertStatesImage from "./images/alert_states.png";
+
+const ButtonSpecs = () => {
+  return (
+    <DxcStack gutter="xxxlarge">
+      <DxcStack gutter="large">
+        <HeadingLink level={2}>Specifications</HeadingLink>
+        <Image src={AlertSpecsImage} alt="alert specifications" />
+        <HeadingLink level={3}>States</HeadingLink>
+        <DxcText as="p">
+          States for the close action of the alert component: <strong>Enabled</strong>,{" "}
+          <strong>hover</strong>, <strong>focus</strong>, and <strong>active</strong>.
+        </DxcText>
+        <Figure caption="Alert action specs">
+          <Image src={AlertStatesImage} alt="Alert action specs" />
+        </Figure>
+      </DxcStack>
+      <DxcStack gutter="large">
+        <HeadingLink level={3}>Anatomy</HeadingLink>
+        <Image src={AlertAnatomyImage} alt="Alert component anatomy" />
+        <DxcList type="number">
+          <DxcText>Container</DxcText>
+          <DxcText>Icon</DxcText>
+          <DxcText>Title</DxcText>
+          <DxcText>Inline text</DxcText>
+          <DxcText>Close action</DxcText>
+          <DxcText>
+            Content <em>(Optional)</em>
+          </DxcText>
+        </DxcList>
+      </DxcStack>
+      <DxcStack gutter="large">
+        <HeadingLink level={3}>Content</HeadingLink>
```

Comment:
I am also not sure about this section but it is next to the anatomy since they are related.
I was thinking of even making it a sub-section of the Anatomy.
I also see the point in having this in Usage instead of specs.

Reply:
I think we should ask @aweell as we both are not sure about this, I see the relation with Anatomy but it's also talking about the correct usage of the component, so I'm a bit confused.

Reply:
The "content" section should be under the usage tab instead of specs